examples/hybrid/driver.jl

```diff
@@ -119,10 +120,41 @@ end
 
 if config.parsed_args["check_conservation"]
     FT = Spaces.undertype(axes(sol.u[end].c.ρ))
-    @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
-    @test sum(sol.u[1].c.ρe_tot) +
-          (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
-          sum(sol.u[end].c.ρe_tot) rtol = 100 * eps(FT)
+
+    # energy
+    energy_total = sum(sol.u[end].c.ρe_tot) - p.net_energy_flux_toa[][]
```

Comment:
```suggestion
    energy_total = sum(sol.u[end].c.ρe_tot)
```
This probably doesn't matter as energy_total is only used in the denominator, but I think it is slightly more clear if the energy_total is just defined as the total energy of the system at the end (or beginning) of the simulation.

Reply:
Sounds good. The `energy_total` is now the energy of the atmosphere + energy of the surfaces (i.e. the whole system) for the prognostic surface.